layouts/ucf-post-list-card.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * The default functions for the card layout
+ **/
+if ( ! function_exists( 'ucf_post_list_display_card_before' ) ) {
+	function ucf_post_list_display_card_before( $posts, $list_title ) {
+		ob_start();
+	?>
+		<div class="ucf-post-list card-layout">
+	<?php
+		echo ob_get_clean();
+	}
+
+	add_action( 'ucf_post_list_display_card_before', 'ucf_post_list_display_card_before', 10, 2 );
+}
+
+
+if ( !function_exists( 'ucf_post_list_display_card_title' ) ) {
+
+	function ucf_post_list_display_card_title( $posts, $list_title ) {
+
+		if ( $list_title ) {
+			$formatted_title = '<h2 class="ucf-post-list-title">' . $list_title . '</h2>';
+		}
+
+		echo $formatted_title;
+	}
+
+	add_action( 'ucf_post_list_display_card_title', 'ucf_post_list_display_card_title', 10, 2 );
+
+}
+
+if ( ! function_exists( 'ucf_post_list_display_card' ) ) {
+	function ucf_post_list_display_card( $posts, $show_image, $posts_per_row, $list_title ) {
+		if ( ! is_array( $posts ) ) { $posts = array( $posts ); }
+		ob_start();
+?>
+		<?php if ( $posts ): ?>
+		<div class="ucf-post-list-card-deck">
+
+		<?php foreach( $posts as $index=>$item ) :
+			$date = date("M d",strtotime($item->post_date));
```

Comment:
Formatting stuff... should be `date( "M d", strtottime( $item->post_date ) );` Add them spaces!